### PR TITLE
docs: update feature development walkthrough

### PR DESCRIPTION
**Pull request by scarlettperry**
### Description
PR updates [Feature Development](https://clutch.sh/docs/development/feature/) docs/examples of the Amiibo workflow.
* fixes the boilerplate frontend code
* fixes the example components
* adds updated screenshots

### Testing Performed
Locally
Used `optipng` to optimize the 3 images.
Can preview the new changes here: https://deploy-preview-896--clutch-preview.netlify.app/docs/development/feature

### TODOs
- [x] update the screenshots

### CODE REVIEW

File: docs/development/feature.md [hunk 1]
```diff
@@ -568,7 +578,8 @@ const register = (): WorkflowConfiguration => {
     displayName: "Amiibo",
     routes: {
       landing: {
-        path: "/",
+  // highlight-next-line
```
Comment (scarlettperry) [inline docs/development/feature.md:581]: https://clutch.sh/docs/development/feature/#test-1 says to navigate to `http://localhost:3000/amiibo/lookup`

File: examples/amiibo/frontend/workflows/amiibo/src/hello-world.tsx [hunk 2]
```diff
@@ -2,16 +2,20 @@ import React, { ChangeEvent } from "react";
 import _ from "lodash";
 import {
   Button,
+  ButtonGroup,
   client,
   useWizardContext,
   Table,
-  Row,
+  TableRow,
```
Comment (scarlettperry) [inline examples/amiibo/frontend/workflows/amiibo/src/hello-world.tsx:9]: component updated/renamed in the redesign

File: docs/development/feature.md [hunk 3]
```diff
@@ -427,13 +427,17 @@ Define a functional component for the amiibo lookup by adding the highlighted li
...
 import {
   Button,
+  ButtonGroup,
   useWizardContext,
   TextField,
 } from "@clutch-sh/core";
 
 import { useDataLayout } from "@clutch-sh/data-layout";
 // highlight-end
+import type { WizardChild } from "@clutch-sh/wizard";
```
Comment (scarlettperry) [inline docs/development/feature.md:437]: these 2 import additions get added as part of the scaffolding

File: examples/amiibo/frontend/workflows/amiibo/src/index.tsx [hunk 4]
```diff
@@ -15,9 +15,9 @@ const register = (): WorkflowConfiguration => {
     displayName: "Amiibo",
     routes: {
       landing: {
-        path: "/",
+        path: "/lookup",
```
Comment (scarlettperry) [inline examples/amiibo/frontend/workflows/amiibo/src/index.tsx:18]: updated b/c docs say to navigate to http://localhost:3000/amiibo/lookup

https://clutch.sh/docs/development/feature/#test-1

**Review by dschaller — APPROVED** [review]

**Pull request merged**